zerver/management/commands/generate_realm_creation_link.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from typing import Any
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.core.urlresolvers import reverse
+from confirmation.models import RealmCreationKey, generate_key
+
+class Command(BaseCommand):
+    help = """Generate link for creating Organization.
+
+Usage: python manage.py generate_realm_creation_link """
```

Comment:
We should expand this help to be more clear about how this feature works:

E.g.

```This generates and outputs a randomly generated, 1-time-use special link.  Whoever visits the link can create a new Zulip realm on this server, regardless of whether settings.OPEN_REALM_CREATION is enabled.```